src/panels/lovelace/cards/hui-history-graph-card.ts

```diff
@@ -149,13 +152,18 @@ export class HuiHistoryGraphCard extends LitElement implements LovelaceCard {
   }
 
   private async _getStateHistory(): Promise<void> {
+    if (this._fetching) {
+      return;
+    }
+    this._fetching = true;
     this._stateHistory = await getRecentWithCache(
       this.hass!,
       this._cacheConfig!.cacheKey,
       this._cacheConfig!,
       this.hass!.localize,
       this.hass!.language
     );
+    this._fetching = false;
```

Comment:
We should put this in a `try…finally{}` block in case `getRecentWithCache` raises?